src/molecule/data/validate-dockerfile.yml

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env ansible-playbook
 ---
 - hosts: localhost
+  collections:
+    - community.docker
```

Comment:
btw, I may be missing something but shouldn't all these docker-related changes be in a different PR ?

Reply:
It is linked, once I added ansible-lint as a direct dependency and moved ansibe-base, I ended up with dockerfile build failing due to missing docker_image module.

Still, I will try to do the changes around that in a preparatory change, so we make this PR smaller. There is still more work to be done on this PR.